app/views/homelesses/_modalphone.html.erb

```diff
@@ -1,8 +1,6 @@
 <div class='homeless_phone' >
-<p> No one Phone registered yet. However map is now revealed.</p>
-<p> Cross <span class="type-rise"> <%= @homeless.name %> </span> according to the last 5 posiiton markers revealed on the map. Blue marker is the last. </p>
+<p> No one Phone registered yet. However the profile's map is now revealed.</p>
```

Comment:
"No one Phone registered yet" sounds strange to me. Maybe change to "No phone registered yet" ?
Is what you mean ?